container-images/local-dev/README.adoc

```diff
@@ -9,5 +9,25 @@ If you are working on improvements to the image however, you can build it locall
 [source,bash]
 ----
 cd uncontained.io/
-docker build -f container-images/local-dev/Dockerfile -t uncontained-local-dev .
+buildah build -f container-images/local-dev/Dockerfile -t uncontained-local-dev .
```

Comment:
shouldn't this be `buildah build-using-dockerfile`?

Reply:
Yes, sorry, that was VisualCode - auto replaced bud to build.